target/linux/realtek/files-5.10/drivers/net/phy/rtl83xx-phy.c

```diff
@@ -2927,6 +3075,13 @@ sds_config rtl9300_a_sds_10gr_lane1[] =
 	{0x2B, 0x14, 0x3108}, {0x2D, 0x13, 0x3C87}, {0x2D, 0x14, 0x1808},
 };
 
+/* TODO: Add patch for USXGMII on SDS 6-8 for AQR113C */
+void rtl9300_sds_patch(int sds_num, sds_config c[], int entries)
+{
+	for (int i = 0; i < entries; ++i)
```

Comment:
int i;
then remove the int from for loop

Reply:
actually, modern kernel development prefers he int loops :) (but only since 5.17)